src/tests/unit/tests/electron/platform/android/rule-information-provider.test.ts

```diff
@@ -16,10 +16,20 @@ describe('RuleInformationProvider', () => {
 
     function buildTouchSizeWcagRuleResultObject(status: string, dpi: number, height: number, width: number): RuleResultsData {
         const props = {};
-        // This is based on the output of the Android service
+        // This is based on the output of the Android service.
         props['Screen Dots Per Inch'] = dpi;
-        props['height'] = height;
-        props['width'] = width;
+
+        const left = 123 as number;
```

Comment:
Pertaining to my comment on the actual test; does `'123' as number` compile? Just curious.

Reply:
No. '123' as number generates an error.